recipes/ffmpeg-normalize/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "ffmpeg-normalize" %}
+{% set version = "1.22.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slhck/ffmpeg-normalize/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3c2fa6c37d91430d2cf2b52d45f450c8795a479cbad43961deece66f0542c136
+
+build:
+  skip: true  # [py<36]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - ffmpeg-normalize = ffmpeg_normalize.__main__:main
+
+requirements:
+  build:
+    - python
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - tqdm >=4.38.0
+    - colorama >=0.4.3
+    - ffmpeg-progress-yield >=0.1.2
```

Comment:
It looks like `ffmpeg-progress-yield` isn't available via the conda-forge channel. You'll need to add it to get this to build without errors.

Reply:
I will create a separate PR for this. Thanks for letting me know.